test/__tests__/data/klaviyo.json

```diff
@@ -658,6 +818,129 @@
       "files": {}
     }
   },
+  {
+    "description": "Track event call with flatten properties enabled",
```

Comment:
Add test case for one ecom event too `Track **ecom** event call with flatten properties enabled`

Reply:
Added